src/webportal/src/app/vc/vc-modal-component.ejs

```diff
@@ -0,0 +1,61 @@
+<!--  Modal component for vc add -->
+  <div class="modal fade vc-model" id="virtualClustersList" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+     <div class="modal-dialog">
+       <div class="modal-content">
+         <div class="modal-header">
+           <button type="button" class="close" data-dismiss="modal" aria-hidden="true">
```

Comment:
Don't hide the close button in ARIA level if there is not another sensible "Close" or "Cancel" button.